### PR TITLE
feat: add images to content selector

### PR DESCRIPTION
**Pull request by karrui**
Already compressed the images. Didn't resize because the text in the images got fuzzy.

## Solution

This PR enhances the Component Selector in the Page Editor by adding visual previews for various block types.

**Breaking Changes**

- [x] No - this PR is backwards compatible

**Features**:

- Added preview images for different block types in the Component Selector
- Implemented a new PopoverContent layout to display block preview images
- Integrated Next.js Image component for optimized image loading

**Improvements**:

- Enhanced user experience by providing visual representations of block types
- Updated PopoverContent to include both image preview and block description
- Refined the layout and styling of the Component Selector popover

### CODE REVIEW

**Comment by karrui** [comment]
* **#682** <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/682?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @karrui and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/studio/src/components/PageEditor/constants.ts [hunk 1]
```diff
@@ -174,57 +174,67 @@ export const BLOCK_TO_META: Record<
...
     description: "Display content in expandable accordions",
     usageText:
       "Accordions hide details by default, so they are great for content that isn't relevant to every reader.",
+    imageSrc: "/assets/block-images/Accordion.png",
   },
   infocards: {
     label: "Cards",
     description: `Link information in "cards" with or without images`,
     usageText: "This block supports up to six cards.",
+    imageSrc: "/assets/block-images/Infocards.png",
```
[the remark this answers is not in the document]
Comment (karrui) [reply apps/studio/src/components/PageEditor/constants.ts:231]: whoops. thx

File: apps/studio/public/assets/block-images/Divider.png
Comment (seaerchin) [inline apps/studio/public/assets/block-images/Divider.png:1]: couldn't find a call site in this pr but ok with keeping this

File: apps/studio/public/assets/block-images/Table.png
Comment (seaerchin) [inline apps/studio/public/assets/block-images/Table.png:1]: also couldn't find htis

**Review by seaerchin — APPROVED** [review]
lgtm, dk why got 2 extra but ok w keeping it in

File: apps/studio/public/assets/block-images/Divider.png
Comment (karrui) [reply apps/studio/public/assets/block-images/Divider.png:1]: ya these two not an available block yet. sehyun just created it for now haha

**Pull request merged**